templates/repo/tag/list.tmpl

```diff
@@ -2,7 +2,7 @@
 
 <div role="main" aria-label="{{.Title}}" class="page-content repository tags">
 	{{template "repo/header" .}}
-	<div class="ui container">
+	<div class="ui container releases-page">
```

Comment:
Indeed .... this is not a "releases-page" 

And it seems redundant to use "ui container releases-page" inside "page-content repository releases"

Reply:
IIRC this was needed to match both the tags and release page, but I guess the selector could be altered to match via two selectors.

Reply:
Problem is the icon in the repo header also matches because technically it is a descendant of `.repository.release`. So if I fix one icon with relative position, it breaks the other.

<img width="296" alt="image" src="https://user-images.githubusercontent.com/115237/236705399-4078f328-1e13-4511-8080-c49bbd979550.png">

Reply:
Fixed, see below.